include/nlohmann/json.hpp

```diff
@@ -2356,7 +2356,7 @@ class basic_json // NOLINT(cppcoreguidelines-special-member-functions,hicpp-spec
         // value only works for objects
         if (JSON_HEDLEY_LIKELY(is_object()))
         {
-            // if pointer resolves a value, return it or use default value
+            // if the pointer resolves a value, return it or use the default value
```

Comment:
If the pointer resolves to a value, return it. Otherwise, return 'default_value'.